src/Deeptable.ts

```diff
@@ -173,6 +173,10 @@ export class Deeptable {
     });
   }
 
+  get tiles(): Tile[] {
+    return this.map(d => d);
+  }
+
```

Comment:
We're going to change this to a list of tiles -- for future facing it will be important to handle undefined.

```suggestion
  get tiles(): (Tile | undefined)[] {
    return this.map(d => d);
  }

```